task/08-objects-tasks.js

```diff
@@ -107,33 +114,72 @@ function fromJSON(proto, json) {
  */
 
 const cssSelectorBuilder = {
+    result: '',
 
     element: function(value) {
-        throw new Error('Not implemented');
+        this.error(1);
+        let obj = Object.create(cssSelectorBuilder);
```

Comment:
Looks alright. But Please implement according to task description -
**Provided cssSelectorBuilder should be used as facade only to create your own class/classes,**